internal/controller/datadogagent/controller_test.go

```diff
@@ -14,20 +14,9 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-func newRequest(ns, name string) reconcile.Request {
```

Comment:
I think we can delete this file entirely 🤔 Will do this in a separate PR though

Reply:
Makes sense, was indeed the only function used